lib/actions/google/drive/sheets/google_sheets.js

```diff
@@ -88,7 +88,10 @@ class GoogleSheetsAction extends google_drive_1.GoogleDriveAction {
                     description: "Should this action attempt to overwrite an existing file",
                     label: "Overwrite Existing Files",
                     name: "overwrite",
-                    options: [{ name: "yes", label: "Yes" }, { name: "no", label: "No" }],
+                    options: [{ name: "yes", label: "Yes" }, {
```

Comment:
super not blocking, but why did the format change?

Reply:
must have been a lint fix thing, I didn't change it